examples/standalone/benchmarks/plotting.py

```diff
@@ -76,6 +76,11 @@
             specific = [x for x in alldata if x["setup"]["version"] == backend]
             if specific:
                 for timer in plot_config["timers"]:
+                    label = None
```

Comment:
I think stripping the plot down to only the most relevant components makes a lot of sense